app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt

```diff
@@ -187,26 +182,91 @@ private fun PlantDetailsContent(
     plant: Plant,
     isPlanted: Boolean,
     onNamePosition: (Float) -> Unit,
-    callbacks: PlantDetailsCallbacks,
+    onFabClick: () -> Unit,
     contentAlpha: State<Float>
 ) {
     Column(Modifier.verticalScroll(scrollState)) {
-        PlantImageHeader(
-            scrollState, plant.imageUrl, callbacks.onFabClick, isPlanted,
-            Modifier.graphicsLayer(contentAlpha.value)
-        )
-        PlantInformation(
-            name = plant.name,
-            wateringInterval = plant.wateringInterval,
-            description = plant.description,
-            onNamePosition = { onNamePosition(it) },
-            toolbarState = toolbarState
+        ConstraintLayout {
```

Comment:
This could easily be a Box + Column?

Reply:
I need to anchor the Fab to the center of the bottom of the image, need a `CL` to avoid the craziness of the previous implementation

Reply:
Ah makes sense!